### PR TITLE
Update MAC_address wikipedia URL in comments

### PR DESCRIPTION
**Pull request by 180909**

### CODE REVIEW

**Review by MaxwellDupre — APPROVED** [review]
Better

File: Lib/uuid.py [hunk 1]
```diff
@@ -608,7 +608,7 @@ def _random_getnode():
     # significant bit of the first octet".  This works out to be the 41st bit
     # counting from 1 being the least significant bit, or 1<<40.
     #
-    # See https://en.wikipedia.org/wiki/MAC_address#Unicast_vs._multicast
+    # See https://en.wikipedia.org/wiki/MAC_address#Universal_vs._local_(U/L_bit)
```
Comment (hauntsaninja) [inline Lib/uuid.py]: Seems like this should be https://en.wikipedia.org/wiki/MAC_address#Unicast_vs._multicast_(I/G_bit)

**Review by hauntsaninja — APPROVED** [review]
cc @JelleZijlstra

File: Lib/uuid.py [hunk 2]
```diff
@@ -401,7 +401,7 @@ def _get_command_stdout(command, *args):
 # over locally administered ones since the former are globally unique, but
 # we'll return the first of the latter found if that's all the machine has.
 #
-# See https://en.wikipedia.org/wiki/MAC_address#Universal_vs._local
+# See https://en.wikipedia.org/wiki/MAC_address#Universal_vs._local_(U/L_bit)
```
Comment (JelleZijlstra) [inline Lib/uuid.py:404]: Feels a little unmaintainable, maybe use a permalink instead: https://en.wikipedia.org/w/index.php?title=MAC_address&oldid=1128764812#Universal_vs._local_(U/L_bit)

**Pull request merged**